.gitignore

```diff
@@ -0,0 +1 @@
+.DS_store
```

Comment:
add .golangci.yaml in the project, and in .gitignore

Reply:
added